docs/contribute-docs.md

```diff
@@ -30,26 +30,47 @@ To build the MyST guide documentation:
 
 This will build the documentation locally so that you can preview what your changes will look like.
 
-## How this relates to the documentation at mystmd.org
+## How to update the content at mystmd.org
+
+When you change the content here or in most other MyST repositories, it will _not_ automatically update mystmd.org. You must take one extra step, described below.
+
+The content at mystmd.org is automatically updated with [a GitHub Action to deploy to our Vercel project](https://github.com/jupyter-book/mystmd.org/blob/main/.github/workflows/deploy.yml). When that action is run, it will pull the latest docs content from several MyST repositories, and push them to our Vercel deployment.
```

Comment:
```suggestion
The content at mystmd.org is updated with [a GitHub Action to deploy to our Vercel project](https://github.com/jupyter-book/mystmd.org/blob/main/.github/workflows/deploy.yml). When that action is run, it will pull the latest docs content from several MyST repositories, and push them to our Vercel deployment.
```

Too confusing to use "automatically" here, since you just stated that the process is not automatic. This is a different automatic, I get it, but still confusing.